common/src/main/java/net/william278/huskhomes/redis/redisdata/RedisPubSub.java

```diff
@@ -0,0 +1,58 @@
+package net.william278.huskhomes.redis.redisdata;
+
+import io.lettuce.core.pubsub.RedisPubSubListener;
+
+public abstract class RedisPubSub<K,V> implements RedisPubSubListener<K,V> {
+    public abstract void message(K channel, V message);
+
+    /**
+     * Message received from a pattern subscription.
+     *
+     * @param pattern Pattern
+     * @param channel Channel
+     * @param message Message
+     */
+    public void message(K pattern, K channel, V message){
```

Comment:
These should be annotated with `@Override` -- and JD comments shouldn't be copied over.